publications.Rmd

```diff
@@ -132,7 +132,7 @@ biorxiv_software <- rbind(
   c(doi = "10.1101/2022.04.14.488419", software = "https://github.com/MarioniLab/SageNet"),
   c(doi = "10.1101/2022.03.08.483466", software = "https://github.com/schmeing/gapless"),
   c(doi = "10.12688/f1000research.73600.2", software = "https://bioconductor.org/packages/scDblFinder"),
-  c(doi = "10.12688/f1000research.73493.1", software = "http://pubassistant.ch/"),
+  c(doi = "10.12688/f1000research.73493.2", software = "http://pubassistant.ch/"),
```

Comment:
should be 10.12688/f1000research.73493.3 as well, no? I get "newer version available" for this one